docs/pages/database-access/reference/configuration.mdx

```diff
@@ -20,7 +20,9 @@ db_service:
 
   # Matchers for registering AWS-hosted databases.
   aws:
-    # Database types, only "rds" is supported currently.
+    # Database types. Valid options are:
+    # 'rds' - discovers and registers AWS RDS and Aurora databases.
+    # 'rds-proxy' - discovers and registers AWS RDS proxy databases.
```

Comment:
Are we missing "redshift" here, or has that docs PR not been merged yet?

Reply:
Also, I think we need an example somewhere of how to configure it statically. Seems like currently "aws" section below is missing these additional fields and explanations of when to set them.

Reply:
will merge redshift changes here